packages/graphql-language-service-server/src/findGraphQLTags.ts

```diff
@@ -69,6 +70,37 @@ const BABEL_PLUGINS: ParserPlugin[] = [
   'logicalAssignment',
 ];
 
+function parseVueSFC(
+  source: string,
+):
+  | { type: 'error'; errors: Error[] }
+  | {
```

Comment:
perhaps it might be easier and more readable to declare this return type union above rather than inline?